src/test/java/io/airlift/slice/TestSliceUtf8.java

```diff
@@ -116,6 +116,13 @@
         // min and max surrogate characters
         invalidSequences.add(new byte[] {(byte) 0b11101101, (byte) 0xA0, (byte) 0x80});
         invalidSequences.add(new byte[] {(byte) 0b11101101, (byte) 0xBF, (byte) 0xBF});
+
+        // check overlong encoding
+        invalidSequences.add(new byte[] {(byte) 0b11000000, (byte) 0xAF});
```

Comment:
I would add a comment at the end of this line:
`// 2-byte encoding of /`
Additionally, I would use something like `A` instead of `/` so that the comment I just added is easier to read.